parsers/svgo/svgo.spec.ts

```diff
@@ -259,4 +258,13 @@ describe('Optimize vector', () => {
       return;
     });
   })
+  describe('Handle no svg vector', () => {
+    it('PDF vector must output an url', async () => {
```

Comment:
We should test two things:
- The property `url` exist and is `string` in the case of `pdf`
- The property `content` exist and is `string`, moreover test that the property `url` doesn't exist on SVG

We linked the seeds to a [Specific figma file](https://www.figma.com/file/9KvLO7F8VPrJ7GxGBWwCr9/Specify---Seeds?node-id=0%3A1). We will add export in pdf in the seed file instead of adding seed manually. Due to that point, we shouldn't test too much specific strings like this one which includes an `id`